docker-compose.yml

```diff
@@ -38,15 +39,14 @@ services:
       - redis
     container_name: donor_reporting_portal_backend
     labels:
-      - traefik.backend=backend
-      - traefik.frontend.rule=PathPrefix:/api/,/admin/,/static/,/sociallogin/,/socialcomplete/
-      - traefik.priority=2
+      - "traefik.http.routers.backend.rule=PathPrefix(`/api/`,`/admin/`,`/static/`,`/sociallogin/`,`/socialcomplete/`)"
+      - traefik.http.routers.backend.service=backend
+      - traefik.http.services.backend.loadBalancer.server.port=8080
       - traefik.enable=true
-      - traefik.port=8000
 
   proxy:
-    image: traefik:v1.7
-    command: --api --docker
+    image: traefik:2.1
+    command: --api --providers.docker
```

Comment:
This needed command needed to be tweaked

Reply:
ohhh thx